SIT/features/06_partition_range/cdm.txt

```diff
@@ -1,2 +1,4 @@
-migrateData com.datastax.cdm.job.MigratePartitionsFromFile migrate.properties
-validateData com.datastax.cdm.job.DiffPartitionsFromFile migrate.properties
+migrateDataDefault com.datastax.cdm.job.Migrate migrate.properties
+validateDataDefault com.datastax.cdm.job.DiffData migrate.properties
```

Comment:
are these default migrate/validation required for partition range specific SIT?

Reply:
@faizalrub-datastax The defaults are testing full migration & validation & do not require any partition-range files.